# üß¨ Web server & Database <!-- omit from toc -->

This section highlights the latest web server and databases for drug design.

![Tools](../assets/tools.webp)

## üìö Table of Contents <!-- omit from toc -->

- [Web Server](#web-server)
  - [Integrated Platform](#integrated-platform)
  - [Molecular generation](#molecular-generation)
  - [Molecular property prediction](#molecular-property-prediction)
  - [Molecular optimization](#molecular-optimization)
  - [Structural Biology and Protein Topology](#structural-biology-and-protein-topology)
- [Database](#database)
  - [üîΩ NAR relavent database - 2023](#-nar-relavent-database---2023)

## Web Server

### Integrated Platform

- **DrugFlow: An AI-Driven One-Stop Platform for Innovative Drug Discovery** [2024]

  Chao Shen, Jianfei Song, Chang-Yu Hsieh, Dongsheng Cao, Yu Kang, Wenling Ye, Zhenxing Wu, Jike Wang, Odin Zhang, Xujun Zhang, Hao Zeng, Heng Cai, Yu Chen, Linkang Chen, Hao Luo, Xinda Zhao, Tianye Jian, Tong Chen, Dejun Jiang, Mingyang Wang, Qing Ye, Jialu Wu, Hongyan Du, Hui Shi, Yafeng Deng, and Tingjun Hou

  [J Chem Inf Model (2024)](https://doi.org/10.1021/acs.jcim.4c00621) | [Server](https://drugflow.com/)

- **WeMol - AI ËçØÁâ©ËÆæËÆ°ËΩØ‰ª∂Âπ≥Âè∞**

  ‰∏≠Â§ßÂîØ‰ø°¬∑ÂîØ‰ø°ËÆ°ÁÆó

  [Home](https://www.wecomput.com/) | [Server](https://wemol.wecomput.com/)

### Molecular generation

- **Reinvent 4: Modern AI‚Äìdriven generative molecule design** [2024]

  Hannes H. Loeffler, Jiazhen He, Alessandro Tibo, Jon Paul Janet, Alexey Voronov, Lewis H. Mervin & Ola Engkvist

  [J Cheminform (2024)](https://doi.org/10.1186/s13321-024-00812-5) | [Code](https://github.com/MolecularAI/REINVENT4)

- **ChemoDOTS: a web server to design chemistry-driven focused libraries** [2024]

  Laurent Hoffer, Guillaume Charifi-Hoareau, Sarah Barelier, St√©phane Betzi, Thomas Miller, Xavier Morelli, Philippe Roche

  [Nucleic Acids Res (2024)](https://doi.org/10.1093/nar/gkae326) | [Server](https://chemodots.marseille.inserm.fr/)

- **The Damietta Server: a comprehensive protein design toolkit** [2024]

  Iwan Grin, Kateryna Maksymenko, Tobias W√∂rtwein, Mohammad ElGamacy

  [Nucleic Acids Res (2024)](https://doi.org/10.1093/nar/gkae297) | [Server](https://damietta.de/)

- **ACFIS 2.0: an improved web-server for fragment-based drug discovery via a dynamic screening strategy** [2024]

  Xing-Xing Shi, Zhi-Zheng Wang, Fan Wang, Ge-Fei Hao, Guang-Fu Yang

  [Nucleic Acids Res (2023)](https://doi.org/10.1093/nar/gkad348) | [Server](http://chemyang.ccnu.edu.cn/ccb/server/ACFIS2/)

### Molecular property prediction

- **ADMETlab 3.0: an updated comprehensive online ADMET prediction platform enhanced with broader coverage, improved performance, API functionality and decision support** [2024]

  Li Fu, Shaohua Shi, Jiacai Yi, Ningning Wang, Yuanhang He, Zhenxing Wu, Jinfu Peng, Youchao Deng, Wenxuan Wang, Chengkun Wu, Aiping Lyu, Xiangxiang Zeng, Wentao Zhao, Tingjun Hou, Dongsheng Cao

  [Nucleic Acids Res (2024)](https://doi.org/10.1093/nar/gkae236) | [Server](https://admetlab3.scbdd.com/)

- **ChemFH: an integrated tool for screening frequent false positives in chemical biology and drug discovery** [2024]

  Shaohua Shi, Li Fu, Jiacai Yi, Ziyi Yang, Xiaochen Zhang, Youchao Deng, Wenxuan Wang, Chengkun Wu, Wentao Zhao, Tingjun Hou, Xiangxiang Zeng, Aiping Lyu, Dongsheng Cao

  [Nucleic Acids Res (2024)](https://doi.org/10.1093/nar/gkae424) | [Server](https://chemfh.scbdd.com/)

- **KinomeMETA: a web platform for kinome-wide polypharmacology profiling with meta-learning** [2024]

  Zhaojun Li, Ning Qu, Jingyi Zhou, Jingjing Sun, Qun Ren, Jingyi Meng, Guangchao Wang, Rongyan Wang, Jin Liu, Yijie Chen, Sulin Zhang, Mingyue Zheng, Xutong Li

  [Nucleic Acids Res (2024)](https://doi.org/10.1093/nar/gkae380) | [Server](https://kinomemeta.alphama.com.cn/)

- **admetSAR3.0: a comprehensive platform for exploration, prediction and optimization of chemical ADMET properties** [2024]

  Yaxin Gu, Zhuohang Yu, Yimeng Wang, Long Chen, Chaofeng Lou, Chen Yang, Weihua Li, Guixia Liu, Yun Tang

  [Nucleic Acids Res (2024)](https://doi.org/10.1093/nar/gkae298) | [Server](http://lmmd.ecust.edu.cn/admetsar3/)

- **ProTox 3.0: a webserver for the prediction of toxicity of chemicals** [2024]

  Priyanka Banerjee, Emanuel Kemmler, Mathias Dunkel, Robert Preissner

  [Nucleic Acids Res (2024)](https://doi.org/10.1093/nar/gkae303) | [Server](https://tox.charite.de/)

- **Deep-PK: deep learning for small molecule pharmacokinetic and toxicity prediction** [2024]

  Yoochan Myung, Alex G C de S√°, David B Ascher

  [Nucleic Acids Res (2024)](https://doi.org/10.1093/nar/gkae254) | [Server](https://biosig.lab.uq.edu.au/deeppk/)

- **PypKa server: online pKa predictions and biomolecular structure preparation with precomputed data from PDB and AlphaFold DB** [2024]

  Pedro B P S Reis, Djork-Arn√© Clevert, Miguel Machuqueiro

  [Nucleic Acids Res (2024)](https://doi.org/10.1093/nar/gkae255) | [Server](https://pypka.org/)

- **BioTransformer 3.0‚Äîa web server for accurately predicting metabolic transformation products** [2024]

  David S Wishart, Siyang Tian, Dana Allen, Eponine Oler, Harrison Peters, Vicki W Lui, Vasuk Gautam, Yannick Djoumbou-Feunang, Russell Greiner, Thomas O Metz

  [Nucleic Acids Res (2024)](https://doi.org/10.1093/nar/gkac313) | [Server](https://biotransformer.ca/)

### Molecular optimization

- **ChemMORT: an automatic ADMET optimization platform using deep learning and multi-objective particle swarm optimization** [2024]

  Jia-Cai Yi, Zi-Yi Yang, Wen-Tao Zhao, Zhi-Jiang Yang, Xiao-Chen Zhang, Cheng-Kun Wu, Ai-Ping Lu, Dong-Sheng Cao

  [Brief Bioinform (2024)](https://doi.org/10.1093/bib/bbae008) | [Server](https://cadd.nscc-tj.cn/deploy/chemmort/)

- **OptADMET: a web-based tool for substructure modifications to improve ADMET properties of lead compounds** [2024]

  Jiacai Yi, Shaohua Shi, Li Fu, Ziyi Yang, Pengfei Nie, Aiping Lu, Chengkun Wu, Yafeng Deng, Changyu Hsieh, Xiangxiang Zeng, Tingjun Hou, Dongsheng Cao

  [Nat Protoc (2024)](https://doi.org/10.1038/s41596-023-00942-4) | [Server](https://cadd.nscc-tj.cn/deploy/optadmet/)

### Structural Biology and Protein Topology

- **AlphaKnot 2.0: a web server for the visualization of proteins‚Äô knotting and a database of knotted AlphaFold-predicted models** [2024]

  Pawel Rubach, Maciej Sikora, Aleksandra I Jarmolinska, Agata P Perlinska, Joanna I Sulkowska

  [Nucleic Acids Res (2024)](https://doi.org/10.1093/nar/gkae443) | [Server](https://alphaknot.cent.uw.edu.pl/)

- **AggreProt: a web server for predicting and engineering aggregation prone regions in proteins** [2024]

  Joan Planas-Iglesias, Simeon Borko, Jan Swiatkowski, Matej Elias, Martin Havlasek, Ondrej Salamon, Ekaterina Grakova, Anton√≠n Kunka, Tomas Martinovic, Jiri Damborsky, Jan Martinovic, David Bednar

  [Nucleic Acids Res (2024)](https://doi.org/10.1093/nar/gkae420) | [Server](https://loschmidt.chemi.muni.cz/aggreprot/)

## Database

- ‚ú® **The 2024 Nucleic Acids Research database issue and the online molecular biology database collection** [2024]

  Daniel J Rigden, Xos√© M Fern√°ndez

  [Nucleic Acids Res (2024)](https://doi.org/10.1093/nar/gkad1173) | [Server](http://www.oxfordjournals.org/nar/database/c/)

- ‚ú® [NAR online Molecular Biology Database Collection](http://www.oxfordjournals.org/nar/database/c/)

### üîΩ NAR relavent database - 2023

| Database name     | URL                                                                                                                                             | Short description                                                                   |
| ----------------- | ----------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------- |
| Withdrawn 2.0     | [Database](https://bioinformatics.charite.de/withdrawn_3/index.php) [Paper](https://doi.org/10.1093/nar/gkad1017)                               | withdrawn drugs with pharmacovigilance data                                         |
| DDBJ 2023         | [Database](https://www.ddbj.nig.ac.jp/) [Paper](https://doi.org/10.1093/nar/gkad1046)                                                           | metabolomics data and associated metadata                                           |
| PLAbDab           | [Database](https://opig.stats.ox.ac.uk/webapps/plabdab/) [Paper](https://doi.org/10.1093/nar/gkad1056) [Code](https://github.com/oxpig/PLAbDab) | reference set of antibody sequences and structures                                  |
| MetaboLights      | [Database](https://www.ebi.ac.uk/metabolights) [Paper](https://doi.org/10.1093/nar/gkad1045) [Code](https://github.com/EBI-Metabolights)        | open data repository for metabolomics                                               |
| ChEMBL 2023       | [Database](https://www.ebi.ac.uk/chembl/) [Paper](https://doi.org/10.1093/nar/gkad1004)                                                         | a drug discovery platform spanning multiple bioactivity data types and time periods |
| AlphaFold DB 2024 | [Database](https://alphafold.ebi.ac.uk/) [Paper](https://doi.org/10.1093/nar/gkad1011)                                                          | structure coverage for over 214 million protein sequences                           |
| INTEDE 2.0        | [Database](http://idrblab.org/intede/) [Paper](https://doi.org/10.1093/nar/gkad1013)                                                            | the metabolic roadmap of drugs                                                      |
| DescribePROT 2023 | [Database](http://biomine.cs.vcu.edu/servers/DESCRIBEPROT/) [Paper](https://doi.org/10.1093/nar/gkad985)                                        | amino acid-level descriptors of protein structures and functions                    |
| ADCdb             | [Database](https://idrblab.org/adcdb/) [Paper](https://doi.org/10.1093/nar/gkad831)                                                             | antibody‚Äìdrug conjugates                                                            |
| PharmGWAS         | [Database](https://ngdc.cncb.ac.cn/pharmgwas) [Paper](https://doi.org/10.1093/nar/gkad832)                                                      | GWAS (genome-wide association studies)-based knowledgebase for drug repurposing     |
| VARIDT 3.0        | [Database](https://idrblab.org/varidt/) [Paper](https://doi.org/10.1093/nar/gkad818)                                                            | phenotypic and regulatory variability of drug transporter (DT)                      |
| TTD               | [Database](https://idrblab.org/ttd/) [Paper](https://doi.org/10.1093/nar/gkad751)                                                               | target druggability information                                                     |
| BioLiP2           | [Database](https://zhanggroup.org/BioLiP) [Paper](https://doi.org/10.1093/nar/gkad630)                                                          | biologically relevant ligand‚Äìprotein interactions                                   |
